src/coreComponents/constitutive/fluid/multifluid/CO2Brine/CO2BrineFluid.cpp

```diff
@@ -231,13 +237,27 @@ void CO2BrineFluid< PHASE1, PHASE2, FLASH >::postProcessInput()
   string const expectedGasPhaseNames[] = { "CO2", "co2", "gas", "Gas" };
   m_p2Index = PVTFunctionHelpers::findName( m_phaseNames, expectedGasPhaseNames, viewKeyStruct::phaseNamesString() );
 
-  createPVTModels();
+  bool isClone = true;
+  if( getParent().getName() == "Constitutive" )
+  {
+    string sectionOutput;
+    string sectionName = "Section : PVT Table generation";
+
+    sectionOutput = GEOS_FMT( "\n{:=^{}}\n", "=", sectionName.size() + 5 );
+    sectionOutput += GEOS_FMT( "{:^{}}\n", sectionName, sectionName.size() + 5 );
+    sectionOutput += GEOS_FMT( "{:=^{}}\n\n", "=", sectionName.size() + 5 );
+
+    std::cout << sectionOutput;
```

Comment:
Just a reminder to keep that in mind, those sections layout will need to be factorized in a dedicated PR.